lib/simulator-xcode-11.js

```diff
@@ -16,6 +17,13 @@ class SimulatorXcode11 extends SimulatorXcode10 {
     return path.resolve(devRoot,
       'Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/LaunchDaemons');
   }
+
+  /**
+   * @override
+   */
+  async setDarkMode (darkMode = true) {
```

Comment:
I'd rather change this to `setUiTheme (isDark = true)` to avoid unnecessary repetitions
or setDarkMode (isEnabled)